build.gradle

```diff
@@ -248,8 +240,7 @@ test {
     minHeapSize = "1G"
     maxHeapSize = "4G"
 
-
-    if (CI == "true" && CLOUD == "false" ) {
+    if (TEST_VERBOSITY == "minimal") {
```

Comment:
It seemed better to control it explicitly rather than being derived from some combination of CI and CLOUD.  I'm happy to change the names though.